src/BackendStatusBanner/index.jsx

```diff
@@ -7,10 +7,15 @@ class BackendStatusBanner extends React.Component {
     this.state = {
       apiConnectionStatus: 200,
     };
+
+    this.apiUrl = '/assets/course-v1:edX+DemoX+Demo_Course/?page=0&page_size=50&sort=sort&asset_type=';
+    if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
+      this.apiUrl = `/api${this.apiUrl}`;
+    }
```

Comment:
This is a fine workaround for now, but I'll probably change it in my redux PR. We discussed this a bit last week -- ultimately, if we store all our Studio URLs in a separate manifest file, we can import them directly into the `devServer` config, not have to worry about pattern matching the `/api` slug, and use the list of URLs as a kind of contract with Studio.

Reply:
That makes sense and I'd be happy to do that work, but it probably makes sense to do that on top of your redux work. Are you okay if I merge this as it is now and you can copy this block into the reducer when you fix the conflict? Then, I can make a PR off your redux one that creates a separate manifest file with our API urls and import them into the reducer and our webpack dev config.

Reply:
For sure, go for it! Don't sweat the conflicts -- I was planning on just addressing them anyway.